Neos.ContentRepository.Core/Classes/ContentGraphFinder.php

```diff
@@ -22,12 +22,11 @@
 use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceName;
 
 /**
- * A finder for ContentGraphInterface bound to contentStream/Workspace
+ * A finder for a ContentGraph bound to contentStream / workspaceName
  *
- * Userland code should not use this directly. You should get a ContentGraph
- * via ContentRepository::getContentGraph()
+ * The API way of accessing a ContentGraph is via ContentRepository::getContentGraph()
  *
- * @api Not for userland code, only for read access during write operations and in services
+ * @internal User land code should not use this directly.
```

Comment:
Now that the whole override thing is gone from here I would have actually left it API as it's a projection state like the others and it seems weird to have one internal and the rest API in there.

Reply:
Well this is just intermediate step, i thought we want to make the content graph projection first level citizen and thus well soon drop this thing again.

Reply:
Maybe later make api, after we saw how it goes